ankihub/gui/operations/username.py

```diff
@@ -0,0 +1,29 @@
+import aqt
+
+from ... import LOGGER
+from ...addon_ankihub_client import AddonAnkiHubClient as AnkiHubClient
+from ...ankihub_client import AnkiHubHTTPError, AnkiHubRequestException
+from ...gui.operations import AddonQueryOp
+from ...settings import config
+
+
+def _fetch_username_in_background() -> None:
+    username = ""
+    if config.is_logged_in() and not config.username():
+        client = AnkiHubClient()
+        try:
+            username = client.get_user_details()["username"]
+        except (AnkiHubRequestException, AnkiHubHTTPError) as exc:
+            LOGGER.error(f"Failed to fetch username: {exc}")
```

Comment:
:wrench: Let's no use log level "error" here, because then it will be sent to sentry because of Sentry's logging integration.  Fetching the user name can fail when the token is expired and we don't want issues on sentry in this case.
```suggestion
            LOGGER.warning(f"Failed to fetch username: {exc}")
```

We are using a logging level in `_setup_feature_flags`, but feature flags can be fetched without being logged in (which I find a bit weird), so for them it's really an exception when they can't be fetched.